libdevcrypto/gm/sm2/sm2.cpp

```diff
@@ -34,34 +36,39 @@ bool SM2::genKey()
     sm2Group = EC_GROUP_new_by_curve_name(NID_sm2);
     if (!sm2Group)
     {
-        LOG(ERROR) << "Error Of Gain SM2 Group Object";
+        CRYPTO_LOG(ERROR) << "[#CRYPTO::SM2::genKey] Error Of Gain SM2 Group Object";
```

Comment:
CRYPTO_LOG(ERROR) << "[genKey] Error Of Gain SM2 Group Object";  maybe better without ”#”.

Reply:
get it